### PR TITLE
Input

### PR DESCRIPTION
**Pull request by shivanigangadharan**
Input component preview :- 

image

Added standard, outlined, filled and their respective error validation styles.

### CODE REVIEW

File: pages/docs/docs.css [hunk 1]
```diff
@@ -37,4 +37,5 @@ body{
 }
 iframe{
     margin: 4rem 0px 0px 0px;
+    max-height: 400px;
```
Comment (SanjayJatti) [inline pages/docs/docs.css:43]: use max-width: 100% to carbon container. That will make it responsive.

File: components/input.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.input-container{
+    display: flex;
+    gap: 20px;
```
Comment (SanjayJatti) [inline components/input.css]: You can use rem unit instead of px for gap

**Review by SanjayJatti — APPROVED** [review]
Overall code is clean and easily readable. You can add other types of forms or inputs. e.g radio button, checkbox, etc,

**Review by ashishnnnnn — APPROVED** [review]
Code seems pretty much structured :+1:

File: components/input.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    align-items: center;
+    color: var(--darkteal);
+    font-size: 1.2rem;
+}
+.input{
+    outline: none;
+    border:none;
+    color: var(--darkteal);
+    padding: 0.9rem 1rem;
```
Comment (Prabhav-Sharma) [inline components/input.css]: You might want to define global variables in the style-guide to maintain consistency for example 
--space-small:1rem;
--space-xsmall:0.5rem;

**Review by Prabhav-Sharma — CHANGES_REQUESTED** [review]

File: components/input.css (hunk 3 above)
Comment (shivanigangadharan) [reply components/input.css]: Done in commit [324f664](https://github.com/shivanigangadharan/style-ui/pull/11/commits/324f664b12767699c2b4c44ec4583acd5636d9a9).

**Review by Prabhav-Sharma — APPROVED** [review]

File: components/input.css (hunk 2 above)
Comment (shivanigangadharan) [reply components/input.css]: Done in commit [e1a563d](https://github.com/shivanigangadharan/style-ui/pull/11/commits/e1a563d2944056196bd10e35c673ca2516f6f1e0)

File: pages/docs/docs.css (hunk 1 above)
Comment (shivanigangadharan) [reply pages/docs/docs.css:43]: Done in commit [ed0510f](https://github.com/shivanigangadharan/style-ui/pull/11/commits/ed0510fa2ff5d7517f3235adee07843c34c2b2c8)

**Pull request merged**